### PR TITLE
dlgPolynomials: adding r code to the dialog

### PR DESCRIPTION
**Pull request by Lunalo**
Replicate working

### CODE REVIEW

File: instat/dlgPermuteRows.vb [hunk 1]
```diff
@@ -35,4 +49,25 @@ Public Class dlgPermuteRows
 
         End If
     End Sub
+
+    Private Sub txtNumberOfPerColumns_Leave(sender As Object, e As EventArgs) Handles txtNumberOfPerColumns.Leave
```
Comment (dannyparsons) [inline instat/dlgPermuteRows.vb]: Is NumberOfPerColumn an integer? If so then you should replace the text box with a numeric up down.

File: instat/dlgPermuteRows.vb [hunk 2]
```diff
@@ -35,4 +49,25 @@ Public Class dlgPermuteRows
...
+
+            ucrBase.clsRsyntax.AddParameter("n", txtNumberOfPerColumns.Text)
+
+
+
+        End If
+
+    End Sub
+
+    Private Sub txtSetSeed_Leave(sender As Object, e As EventArgs) Handles txtSetSeed.Leave
```
Comment (dannyparsons) [inline instat/dlgPermuteRows.vb]: The same with "seed". It should be an integer (although a non integer will not give an error). To keep it simple I think we use a numeric up down here as well.

**Comment by dannyparsons** [comment]
You've called this pull request dlgPolynomials, but this is not included and there are 2 other dialogs in here. Did you mean to do that?

**Comment by Lunalo** [comment]
Hey Danny,

I will fix it to accommodate integer:
The last one is dlgPermute. Sorry for that

Regards,
John Ong'ala Lunalo|| Kenyatta University Alumnus||
Volunteer, African Maths Initiative||
Email Address :johnlunalo95@gmail.com
Tel: +254701519572, +254786438572

**Comment by Lunalo** [comment]
Hi Danny,

I have checked in r and set.seed() is working with decimals. So, I have
only replaced for "n" which seems to only work with integers

Thanks

Regards,
John Ong'ala Lunalo|| Kenyatta University Alumnus||
Volunteer, African Maths Initiative||
Email Address :johnlunalo95@gmail.com
Tel: +254701519572, +254786438572

**Comment by dannyparsons** [comment]
John, you didn't read my comment carefully:
"It should be an integer (**although a non integer will not give an error**). To keep it simple I think we use a numeric up down here as well."

I didn't say it only works for integers, I said we use make sure it is an integer in the dialog. And the reason is from the R help.
See the R help for set.seed:

> seed: a single value, **interpreted as an integer**

So yes, it does work with decimal numbers, but it is meant to be an integer, and if you give a decimal, it interprets it as an integer. Therefore I think we should only allow integers to be entered.

It's great that you went to R to see how the function works, but you also need to read the help carefully to understand the meaning of each parameter.

**Comment by Lunalo** [comment]
Thanks for that Clarification Danny

On 2/19/16, Danny Parsons <notifications@github.com> wrote:
> John, you didn't read my comment carefully:
> "It should be an integer (**although a non integer will not give an
> error**). To keep it simple I think we use a numeric up down here as well."
>
> I didn't say it only works for integers, I said we use make sure it is an
> integer in the dialog. And the reason is from the R help.
> See the R help for set.seed:
>
>> seed: a single value, **interpreted as an integer**
>
> So yes, it does work with decimal numbers, but it is meant to be an integer,
> and if you give a decimal, it interprets it as an integer. Therefore I think
> we should only allow integers to be entered.
>
> It's great that you went to R to see how the function works, but you also
> need to read the help carefully to understand the meaning of each
> parameter.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/africanmathsinitiative/Instat/pull/872#issuecomment-186170759


-- 
Regards,
John Ong'ala Lunalo|| Kenyatta University Alumnus||
Volunteer, African Maths Initiative||
Email Address :johnlunalo95@gmail.com
Tel: +254701519572, +254786438572

**Comment by Lunalo** [comment]
hi Danny,

I have made those changes.

Thanks

Regards,
John Ong'ala Lunalo|| Kenyatta University Alumnus||
Volunteer, African Maths Initiative||
Email Address :johnlunalo95@gmail.com
Tel: +254701519572, +254786438572

**Pull request merged**